packages/pci-sdk-iframe/src/apiClient/index.ts

```diff
@@ -93,11 +100,16 @@ export async function getCardData(
 		headers,
 		body,
 	})
-		.then((res) => {
-			// TODO: Handle different errros better
```

Comment:
I always liked that the TODO to handle errors better had an error in the word "error". I'm sad to see this one go.